docs/MergingPrs.md

```diff
@@ -0,0 +1,34 @@
+# Merging a PR
+
+**Who can merge?** 
+A PR is to be merged by the author of the PR if the author has commit privileges. 
+Else it should be merged by the main reviewer.  
```

Comment:
Yes, in some cases PM can do it (e.g. if no manual testing is needed). I'll add that.